community/FixEarthLighting.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: page
+title: Fixing lighting issues with Chunky
+parent: Community Guides
+nav_order: 10
+---
+
+Earth maps are usually made with the program WorldPainter, which exports worlds with somewhat broken lighting data.  
```

Comment:
```suggestion
Earth maps are usually created using third party tools such as WorldPainter, which create minimal chunk data, often missing important components such as lighting information.
```

Reply:
Just wanted to clean this up to be a bit more generic (not just WorldPainter, but as an example) and specify that it's not just lighting data but that's what we care about here